Modules/_asynciomodule.c

```diff
@@ -8,6 +8,7 @@
 #include "pycore_runtime_init.h"  // _Py_ID()
 #include "pycore_moduleobject.h"  // _PyModule_GetState()
 #include "structmember.h"         // PyMemberDef
+#include "cpython/context.h"
```

Comment:
Isn't this is already included by `Python.h`?

Reply:
Whoops, sorry about that. There is some plugin in my editor which mysteriously likes to add `#includes` which aren't needed. I've not had success disabling it!